user_guide_src/source/models/model/015.php

```diff
@@ -5,4 +5,11 @@
     'email'    => 'd.vader@theempire.com',
 ];
 
+// Inserts data and returns inserted row's primary key
 $userModel->insert($data);
+
+// Inserts data and returns true on success and false on failure
+$userModel->insert($data, false);
+
+// Returns inserted row's primary key
+$userModel->getInsertID();
```

Comment:
You deleted too much!
We need LF at the end of a file.

Reply:
Thanks I'll check that out.